### PR TITLE
Jaron variable homework2 git

### PR DESCRIPTION
**Pull request by JaronHarness**
Updates to incorrect answers

### CODE REVIEW

File: VariablesHomework/VariablesHomework.cs [hunk 1]
```diff
@@ -7,51 +7,63 @@ public class Variables
...
             Assert.IsTrue(c == 'b'); // Make this pass
+            
+            Assert.IsTrue(c != 'c'); // Then make this pass
 
-            Assert.IsTrue(c == 'c'); // Then make this pass
-
+            
             int z = i;
-            Assert.IsTrue(z == /*Answer here*/); // Make this pass
+            z = 25;
```
Comment (erichamion) [inline VariablesHomework/VariablesHomework.cs]: If you just keep `int z = i` and don't change `z`, then what is `z`'s value?

Comment (JaronHarness) [reply VariablesHomework/VariablesHomework.cs]: 25. because the i = 25; integar at the top of this section

Comment (erichamion) [reply VariablesHomework/VariablesHomework.cs]: Is it? Or has `i` changed since then?

Comment (JaronHarness) [reply VariablesHomework/VariablesHomework.cs]: Oh i = 100;

Comment (erichamion) [reply VariablesHomework/VariablesHomework.cs]: If you stay on this git branch (JaronVariableHomework2_git), you can make your change, then commit and push. This pull request should update with the new commit, without needing a new pull request.

Comment (JaronHarness) [reply VariablesHomework/VariablesHomework.cs]: I think not keeping the comments that showed where the answers would go messed me up quite a bit when making corrections. Are we good to just leave those in and put the answers to the left of the comments from now on?

Comment (erichamion) [reply VariablesHomework/VariablesHomework.cs]: Like `Assert.IsTrue(someVariable == 42 /*Answer here*/);`? That works.

Comment (JaronHarness) [reply VariablesHomework/VariablesHomework.cs]: Yeah

**Review by erichamion — APPROVED** [review]

**Pull request closed without merging**